packages/components/src/SubHeaderBar/SubHeaderBar.component.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../Icon';
+import { Action } from '../Actions';
+import ActionBar from '../ActionBar';
+import theme from './SubHeaderBar.scss';
+
+function DetailsIcon({ iconFile }) {
+	if (iconFile) {
+		return (
+			<span className={theme['subheader-file-icon-box']}>
+				<Icon name={iconFile} className={theme['subheader-file-icon']} />
+			</span>
+		);
+	}
+	return null;
+}
+
+DetailsIcon.propTypes = {
+	iconFile: PropTypes.string,
+};
+
+function DetailsTitle({ title, subTitle, iconFile, onEdit }) {
+	return (
+		<div className={theme['subheader-details']}>
+			<DetailsIcon iconFile={iconFile} />
+			<span className={theme['subheader-details-text']}>
+				<span className={theme['subheader-details-title']}>{title}</span>
```

Comment:
Make sure you add global classes to those classes definitions.

Reply:
I don't understand plainly the global classes. I will study it and make the appropriate change later on.

Reply:
the idea is to add the ability to select or customize things from the outside

Reply:
Yep it's the implem that I don't see, which one are global etc. But  I'm on it :)